platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -202,6 +216,8 @@ public void exportWithString() throws Exception {
         ConfigurationMigrationManager configurationMigrationManager =
                 createConfigurationMigrationManager();
 
+        when(Paths.get("/export/dir")).thenReturn(exportDirectory);
```

Comment:
Might need to use `Paths.get("export", "dir")` if the `dir` folder is inside the `export` one. Letting the `Paths` object build it up is usually better for Windows compatibility.